src/m365/commands/docs.ts

```diff
@@ -0,0 +1,38 @@
+import AnonymousCommand from '../base/AnonymousCommand';
+import { ChildProcess } from 'child_process';
+import { Cli } from '../../cli/Cli';
+import commands from './commands';
+import { Logger } from '../../cli/Logger';
+import * as open from 'open';
+import { settingsNames } from '../../settingsNames';
+const packageJSON = require('../../../package.json');
+
+class DocsCommand extends AnonymousCommand {
+  public get name(): string {
+    return commands.DOCS;
+  }
+
+  public get description(): string {
+    return 'Opens the CLI for Microsoft 365 docs webpage in the default browser';
+  }
+
+  constructor(private open: any) {
+    super();
+  }
+
+  public async commandAction(logger: Logger): Promise<void> {
+    if (Cli.getInstance().getSettingWithDefaultValue<boolean>(settingsNames.autoOpenLinksInBrowser, false) === false) {
+      logger.log(`Use a web browser to open the CLI for Microsoft 365 docs webpage ${packageJSON.homepage}`);
+      return;
+    }
+
+    logger.log(`Opening the CLI for Microsoft 365 docs webpage in the default browser. The URL is ${packageJSON.homepage}`);
+    await this.openBrowser(packageJSON.homepage);
+  }
+
+  private async openBrowser(docsLink: string): Promise<ChildProcess> {
+    return this.open(docsLink);
+  }
+}
+
+module.exports = new DocsCommand(open);
```

Comment:
According to me, it's not really needed to pass `open` as property to the class. We could refactor it to something like 

``` typescript
private async openBrowser(docsLink: string): Promise<void> {
  await open(docsLink);
}
```

Reply:
I made this change and created stub for openBrowser funcition as `openStub = sinon.stub(command as any, 'openBrowser').resolves();` , but it doesn't seem to be covering the function. Would you know what I am missing here?

`it('should open the CLI docs URL using "open"', async () => {
    getSettingWithDefaultValueStub.restore();
    getSettingWithDefaultValueStub = sinon.stub(cli, 'getSettingWithDefaultValue').returns(true);
    await command.action(logger, { options: {} });
    assert(openStub.called);
  });`

Reply:
Hi, @Saurabh7019 you are right I made a small error of judgment here, sorry for that! We can make the test work but not the coverage indeed. I suggest that we just use somewhat the same method as we do in the `cli reconsent` command.